config/application.yml.default

```diff
@@ -90,7 +90,6 @@ doc_auth_s3_request_timeout: 5
 doc_auth_error_dpi_threshold: 290
 doc_auth_error_glare_threshold: 40
 doc_auth_error_sharpness_threshold: 40
-doc_auth_link_sent_controller_enabled: false
```

Comment:
There's another reference to this we can remove in `redo_document_capture_action_spec.rb`:

https://github.com/18F/identity-idp/blob/333d9d554e9810d7c4cf4093f77eb8ebfb5b8755/spec/features/idv/actions/redo_document_capture_action_spec.rb#L7

Reply:
Good catch on both of those, thanks! This snuck in after rebasing on main.